src/store/modules/tasks.js

```diff
@@ -0,0 +1,83 @@
+import tasksApi from "../../api/tasksApi";
+
+const state = () => ({
+    all: [],
```

Comment:
```suggestion
    tasks: [],
```
đặt tên biến dễ hiểu để tiện cho maintain đó em, cái bài tập này đơn giản thì ko sao, sau vào dự án thì cần có thói quen này